custom_dc/README.md

```diff
@@ -0,0 +1,218 @@
+# Custom Data Commons
+
+A custom Data Commons can hold new data and include UI modifications that are
+custom to a third party's need.
+
+This document illustrates development and deployment details for a custom Data
+Commons instance.
+
+## Prerequisit
+
+- Install [docker](https://www.docker.com/products/docker-desktop/)
+- [Optional] Install [gcloud](https://cloud.google.com/sdk/docs/install-sdk)
+  command line tool.
+
+## API Key
+
+- A custom Data Commons needs to connect with main Data Commons. Get API key for
+  Data Commons by sending an email to `support@datacommons.org`.
+
+- [Optional] Provision a Google Maps API key from your GCP project. This is
+  optional and used for place search in visualization tools. Refer to [Maps API
+  Key](TODO) section for detailed instructions.
+
+## Prepare Custom Data
+
+[Note] Skip this section if you don't have custom Data.
+
+Prepare CSV files with statistical data in the following formats:
+
+```csv
+geoId,observationDate,stat_var_1,var_2
+06,2021,555,666
+08,2021,10,10
+```
+
+```csv
+name,observationDate,stat_var_1,stat_var_2
+California,2021,555,666
+Colorado State,2021,10,10
+```
+
+```csv
+dcid,observationDate,stat_var_1,stat_var_2
+geoId/06,2021,555,666
+geoId/08,2021,10,10
+```
+
+The first column header is a property that identifies the observed entity,
+supported properties are `dcid`, `name`, `geoId`. When `dcid` is used, the
+entity should have been resolved (from a previous step).
+
+Refer to [sample folder](./sample) for supported example data files.
+
+Put all the input files under a local folder or Google Cloud Storage folder, and
+use the folder path for environment variable `SQL_DATA_PATH` as described in the
```

Comment:
Can we do the renaming to `CUSTOM_DC_DATA_PATH` in this PR as well?

Reply:
That will touch several files. Maybe leave this as a documentation PR and change them all in a follow up?